README.md

```diff
@@ -119,6 +121,7 @@ const { flags, values, positionals } = parseArgs(argv, options);
 ```
 
 ```js
+const { parseArgs } = require('@pkgjs/parseargs');
```

Comment:
This address one of the linting issues, might as well make it so samples are copy/pastable.